src/util/archive/fuzz_tar.c

```diff
@@ -0,0 +1,89 @@
+#if !FD_HAS_HOSTED
```

Comment:
Looks like there's a good fuzz corpus from go-tar, https://github.com/strongcourage/fuzzing-corpus/tree/master/tar/go-fuzz

Can we use this?